File: JavaProject/Clients.java
import java.util.List;

public class Clients {
	
	private static List<Client> clients;
	

	public static void Register_client(String last_name, String first_name, String client_id, String email){
		Client klient = new Client(last_name, first_name, client_id, email);
		clients.add(klient);
		
	}


	public static List<Client> getClients() {
		return clients;
	}


	public void setClients(List<Client> clients) {
		this.clients = clients;
	}
}

File: CSharpProject/Hotel/Model/Clients.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Hotel.Model
{
    class Clients
    {
        List<Client> allClients;

        public Clients()
        {
            allClients = new List<Client>();
        }

        public void RegisterClient(String firstName, String lastName, String clientID, String email)
        {
            throw new NotImplementedException();
        }

        #region getters and setters
        public Client this[int i]
        {
            get { return allClients[i]; }
            set { allClients[i] = value; }
        }
        #endregion
    }
}
